src/Core/Commander/ManagerCommands.js

```diff
@@ -77,44 +80,35 @@ define('Core/Commander/ManagerCommands', [
             return this.commandsLength()===0;
         };
 
-        ManagerCommands.prototype.runAllCommands = function() {
-
-
-            if (this.commandsLength() === 0)
-            {
-                return Promise.resolve(0);
+        ManagerCommands.prototype.fillCommandPool = function() {
+
+            var command;
+            var that = this;
```

Comment:
I think @tbroyer advised against this pattern and recommended using `bind(this)`